### ISSUE TITLE
Feature Request: Change Streamlabs OBS scene on command and/or channel point reward

### ISSUE THREAD

**Issue by JasonCostanza** [PhantomBot/PhantomBot#2416]
PhantomBot Version: 3.2.2
O/S Version: 20H2
Java Version: No idea
Web Browser Version (if Panel related):

Detailed Description:
I am looking around at trying to enable multiple POV's to appear in my stream by chat or a moderator triggering a command based on chat input or redeeming a channel point reward. I understand Streamlabs Chatbot can do this already and I was really eager to see if phantombot can do this.

We use phantombot because it is the only bot, that we know of, which allows moderators of the channel to access the bot remotely through web browser. We LOVE this feature and is imperative to our channel as our broadcaster isn't up to speed on the tech of bots like myself. I run all her tech for her but I am not at their house to work on the bot while the stream is live.

Steps to Reproduce:
Being a feature request, no steps available.

Logs:

Screenshots (if applicable):

**Comment by robsdedude** [comment]
I'll leave this here: https://github.com/Twasi/websocket-obs-java

**Comment by JasonCostanza** [comment]
> I'll leave this here: https://github.com/Twasi/websocket-obs-java

I thought the websocket thing was for OBS not SLOBS which is what we use to stream with. Correct me if I'm wrong

**Comment by robsdedude** [comment]
Indeed. But if one goes the length of adding such a feature, I thinks it's worth aiming for supporting multiple streaming applications.

SLOBS: https://github.com/stream-labs/streamlabs-obs-api-docs

Couldn't fine something for XSplit quickly.

**Comment by gmt2001** [comment]
https://xjsframework.github.io/quickstart.html#extension
